CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 5.0.0-beta.1 (2021-09-15)
+
+- Migrated to Angular CLI. [#99](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/99)
+- Added support for "partial" Ivy compilation mode. [#99](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/99)
```

Comment:
I think that in a similar changelog for another repo, we changed the quote marks around "partial" to tick marks to format it as a code snippet, but I can't remember where else we made this change.